src/webgpu/api/validation/query_set/create.spec.ts

```diff
@@ -2,140 +2,110 @@ export const description = `
 Tests for validation in createQuerySet.
 `;
 
-import { poptions } from '../../../../common/framework/params_builder.js';
+import { params, poptions } from '../../../../common/framework/params_builder.js';
 import { makeTestGroup } from '../../../../common/framework/test_group.js';
 import { ValidationTest } from '../validation_test.js';
 
 export const g = makeTestGroup(ValidationTest);
 
 const kMaxQueryCount = 8192;
+const kQuerytype = ['occlusion', 'pipeline-statistics', 'timestamp'] as const;
```

Comment:
nit: These two constants can be in capability_info.ts. I pushed this change to your branch now.